doc/api/index.md

```diff
@@ -1318,7 +1318,18 @@ _return value_: ``void``
 
 Deactivate the current video track, if one.
 
-> Will enter in `RELOADING` state for a short period.
+Will enter in `RELOADING` state for a short period, if not in _DirectFile_
```

Comment:
Are we sure that we cannot remove a source buffer while playing?

I know that the opposite is true (we cannot add one) but I'm not sure about that one.

In any case, `Might` is better than `Will` here, as will imply that the `RELOADING` state SHOULD be triggered after disableVideoTrack as per the API.

By just saying `Might` we could just find a way to not reload in the future and the API would not be broken.